eve/workers/pod-example-solution-builder/Dockerfile

```diff
@@ -5,25 +5,25 @@ ARG OPERATOR_SDK_VERSION=v0.16.0
 
 WORKDIR /home/eve/workspace
 
-RUN yum install -y \
+RUN yum install -y --setopt=skip_missing_names_on_install=False \
```

Comment:
Not sure we need this extra check here!
should `yum` not automatically fail if it cannot find any of the packages

Reply:
Hmm, seems yum will not fail when it cannot find a package from the list provided so let's keep it.

Reply:
Yup, that's why I've added this option, the build would have failed here with a clear error message if we had this option set.

Reply:
ok

Reply:
Well... not a clear message because it would have failed when provisioning the worker... but at least it's easily reproducible, just by building this Dockerfile.